src/components/Header.jsx

```diff
@@ -0,0 +1,43 @@
+import { NavLink } from "react-router-dom";
+import "./Header.css";
+
+export const Header = () => {
+  return (
+    <header>
+      <div className="navigation-box">
+        <div className="header-text-box">
+          <h1>The Movie Paradise</h1>
+          <h2>Get inspiration, pop some popcorn and get ready for a movie night</h2>
+        </div>
+        <div className="dropdown-menu">
+          <button>Menu</button>
+          <nav className="dropdown-menu-content">
+            <NavLink to="/top-rated" className="mobile-menu-link">
+              Top rated
+            </NavLink>
+            <NavLink to="/popular" className="mobile-menu-link">
+              Popular
+            </NavLink>
+            <NavLink to="/" className="mobile-menu-link">
+              Home
+            </NavLink>
+          </nav>
+        </div>
```

Comment:
Is it for accessibility the menu need to be able to close when clicking on it or is it just best practice or something else? Because now it will show or hide depending on if you hover over it or not. Just curious why it has to be able to close when clicking on it?